test/jdk/javax/swing/border/TestBevelBorderParam.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @bug 8370467
+ * @summary Verifies createBevelBorder and createSoftBevelBorder does not throw NPE
+ * @run main TestBevelBorderParam
+ */
+
+import javax.swing.BorderFactory;
+import javax.swing.border.BevelBorder;
+
+public class TestBevelBorderParam {
+
+    public static void main(String[] args) throws Exception {
+        StringBuilder str = new StringBuilder();
+
+        try {
+            BorderFactory.createBevelBorder(BevelBorder.RAISED, null, null);
+        } catch (NullPointerException ex) {
+            str.append("\n");
+            str.append("BorderFactory.createBevelBorder throws NPE for null highlight and shadow");
+        }
+
+        try {
+            BorderFactory.createSoftBevelBorder(BevelBorder.RAISED, null, null);
+        } catch (NullPointerException e) {
+            str.append("\n");
+            str.append("BorderFactory.createSoftBevelBorder throws NPE for null highlight and shadow");
+        }
+
+        try {
+            new BevelBorder(BevelBorder.RAISED, null, null);
+        } catch (NullPointerException ex) {
+            str.append("\n");
+            str.append("BevelBorder constructor throws NPE for null highlight and shadow");
+        }
+        if (str.length() != 0) {
```

Comment:
```suggestion
        }

        if (str.length() != 0) {
```

The `if` statement also starts a new logical block.